contracts/PassiveSwap/AbstractSwapper.sol

```diff
@@ -0,0 +1,133 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.13;
+
+import { AccessControlledV8 } from "@venusprotocol/governance-contracts/contracts/Governance/AccessControlledV8.sol";
+import { ResilientOracle } from "@venusprotocol/oracle/contracts/ResilientOracle.sol";
+import { ensureNonzeroAddress } from "../lib/validators.sol";
+import { VTokenInterface } from "../VTokenInterfaces.sol";
+import { MANTISSA_ONE, EXP_SCALE } from "../lib/constants.sol";
+
+contract AbstractSwapper is AccessControlledV8 {
+    /// @notice Swap configuration for the tokens pair
+    struct SwapConfiguration {
+        /// tokenIn address
+        address tokenAddressIn;
+        /// tokenOut address
+        address tokenAddressOut;
+        /// incentive on swapping tokens
+        uint256 incentive;
```

Comment:
I would add a comment to clarify the unit of this variable, and I would give some examples. AFAIU, `10 * 1e18` would be an incentive of 10%, right?

Reply:
No, 10% incentive means 0.1 * 1e18
I have added comment also, d1c6ee3894085fd33e8ac98110d52004eab99ac9